committee-steering/governance/ug-governance.md

```diff
@@ -0,0 +1,38 @@
+# Kubernetes User Group (UG)
+
+## Introduction
+The Kubernetes community and ecosystem benefits providing a place for communication and collaboration between users and community members on topics that either do not have deliverables or are outside the scope of the project itself. These topics are not appropriate for forming SIGs, WGs or Committees, which are intended to drive work within the project. User Groups are the venue for this kind of communication and collaboration.
+
+Example User Groups:
+
+- Big Data - Topics related to Big Data integrations.
+- Cloud Providers - Topics related to public/private cloud provider integrations.
+- Machine Learning - Topics related to ML ecosystem integrations.
+- DevOps - Topics related to DevOps tools and deployments.
+
+## UG Goals
+- Facilitate communication between Kubernetes users and contributors on topics pertaining to - the usage of, the extension of and integration with Kubernetes and Kubernetes subprojects.
+
+## UG Anti-Goals
+- Producing or owning Kubernetes project deliverables or assets.
+- Making Kubernetes project decisions.
+- Creating Kubernetes project roles.
+
+## UG Example Activities
+- Facilitating collaboration between user group members - which may result in the production of - blogs, docs, guides, demos, presentations, prototypes, external contributions, etc. (UG is itself not responsible to create any such content, it merely facilitate members creating them).
+- Anything produced within the context of the user group can be ultimately owned by a SIG as a subproject or is owned by individuals in the user group.
+- Publishing content to a GitHub directory owned by the user group. Access to this location is controlled by OWNERS files. OWNERS files are seeded at user group creation time with the Chairs.
+- Providing updates from the user group’s activities to the community at large.
+  - Cadence - every release.
+  - Improvements/Contributions/Pain-points through the user group to the ecosystem (features enabled, user group specific engagements, etc.)
+
+## Assets Owned by UG
+- Calendar Events
+- Slack Channel
+- Discussion Forum Groups
+- GitHub Directory
```

Comment:
@daminisatya - @pwittrock asked above ... what's the "GitHub directory"?

Reply:
@dims Phil asked me to add GitHub Directory to the assets owned :) 

Not sure why the review comment is not shown as outdated!

Reply:
weird. ack :)

Reply:
This needs a bit more explanation imo.  UGs don't own code.

Reply:
UGs don't own anything, but folks were asking for a place to store documents pertaining to the UG.

Reply:
I am struggling to parse the meaning. I assume it is a "Directory in the Community Github Repo"?

Reply:
@philips Yes, It is a folder dedicated for that particular UG in the community GitHub repo.

Reply:
@pwittrock why can't they use a discuss forum and PR other docs in the community?  

cc: @castrojo